tslib.es6.js

```diff
@@ -354,6 +354,10 @@ export default {
     __rest: __rest,
     __decorate: __decorate,
     __param: __param,
+    __esDecorate: __esDecorate,
+    __runInitializers: __runInitializers,
+    __propKey: __propKey,
+    __setFunctionName: __setFunctionName,

```

Comment:
New tests caught that these were missing

Reply:
I feel like the fact that we got away without these in the .es6 files would imply that nobody's using them... Or just I guess that the overlap of new features and these old files is small...

Reply:
It’s not necessarily that nobody’s using these files; it’s that nobody’s using the default export from these files, which is only there for edge casey compat